src/chai-form-header-configurator.ts

```diff
@@ -0,0 +1,194 @@
+
+    if (shadowRoot) {
+      const inputRanges = shadowRoot.querySelectorAll('input[type="range"]');
+
+      // Loop through each input range
+      inputRanges.forEach((input: Element) => {
+        // Cast the input to HTMLInputElement
+        const htmlInput = input as HTMLInputElement;
+
+        // Get the current value of the input
+        const currentValue = htmlInput.value;
+
+        // Construct the span ID based on the input ID
+        const spanId = `${htmlInput.id}-slider-value`;
+
+        // Get the corresponding span using the constructed ID
+        const fontSizeDisplay = shadowRoot.getElementById(
+          spanId
+        ) as HTMLSpanElement;
+
+        if (fontSizeDisplay) {
+          // Update the span's text content with the current value
+          fontSizeDisplay.textContent = currentValue;
+        } else {
+          console.warn(
+            `Span with ID ${spanId} not found for input ${htmlInput.id}`
+          );
+        }
+      });
+    }
+  }
+
+  initlizeComponentWithDefaults() {
+    // Access the Shadow DOM
+    this.intilizeRangesElementsSapnWidhRangeValue();
+  }
+
+  applyChaiStyle(event: Event, cssVariable: string, px: string) {
+    const target = document.getElementById(this.targetForm) as ChaiForm;
+    if (!target) return;
+
+    const input = event.target as HTMLInputElement;
+    target.style.setProperty(cssVariable, input.value + px);
+
+    // if input is range reflect it's value to element span.
+    if (input.type == 'range') {
+      const shadowRoot = this.shadowRoot;
+      if (shadowRoot) {
+        const sliderValueId = input.id + '-slider-value';
+        var sliderValueElement = shadowRoot.getElementById(
+          sliderValueId
+        ) as HTMLInputElement;
+
+        if (sliderValueElement) {
+          sliderValueElement.innerText = input.value;
+        }
+      }
+    }
+
+    if (cssVariable != 'FormStyleClass') {
+      // exclude this input [FormStyleClass] from addign to css style , becuase i will assign it to defaultClassStyle varaibles
+      // defaultClassStyle variables which idenitify form style class
+      this.cssTyles[cssVariable] = input.value + px;
+    }
+    // fire event which indicate style has been changed so i need to notify snippet component with new style changes.
+    this.notifySnippetComponent();
+  }
+
+  resetFormStyle() {
+    const target = document.getElementById(this.targetForm) as ChaiForm;
+    if (!target) return;
+
+    target.removeAttribute('style');
+    this.cssTyles = {};
+    this.notifySnippetComponent();
+  }
+
+  notifySnippetComponent() {
+    setTimeout(() => {
+      window.dispatchEvent(
+        new CustomEvent('style-updated', {
+          detail: {
+            chaiHeaderstyles: this.cssTyles,
+            defaultClass: this.defaultClassStyle,
+            from: 'chai-header',
+          },
+          bubbles: true,
+          composed: true,
+        })
+      );
+    }, 0);
+  }
+
+  override render() {
```

Comment:
This PR misses the ability to configure several important attributes, e.g. the header text and button text.

Reply:
Hello lars  , 
I created two components for header and button configurator
I just found two css variable for header and some button css variables  in chai-form
and I created configurators based on this css variables which exist in chai-form 
can you clarify for me where can I find the header text and button text retributes.